src/javascripts/cookies.js

```diff
@@ -0,0 +1,43 @@
+// Add cookie to session
+
+(function() {
+  var
+    cookie = false,
+    cookieContent = document.getElementById('cookie');
+
+  checkCookie();
+  if (cookie === true) {
```

Comment:
You could tighten this up - it's display: none by default - could be refactored as:

`if (!cookie) { cookieContent.style.display = 'block' }` without worrying about the else statement here.